### ISSUE TITLE
Self hosting issues

### ISSUE THREAD

**Issue by dabugen** [Swetrix/roadmap#172]
Dear Swetrix Team,

Thank you for creating Swetrix and offering a self-hosted option - I would appreciate the opportunity to evaluate it. I have deployed Swetrix using Docker Compose on Ubuntu, however I am unable to login to the interface regardless of the method attempted.

1)    When using https:// for the API URL, as securing the interface with SSL encryption is preferred, I receive the following error:
![swetrix-ssl](https://github.com/Swetrix/roadmap/assets/22604901/308a65d3-423e-4da8-b146-01aba3e465d5)

2)    When starting on port 80 using HTTP://, a connection is established but login fails with the previously specified credentials in docker-compose.yml, displaying:
![swetrix-firefox](https://github.com/Swetrix/roadmap/assets/22604901/92ff0a1f-4d7c-4863-b4cc-a4e8edd4caff)
![swetrix-chrome](https://github.com/Swetrix/roadmap/assets/22604901/f9eff3f0-69f9-4510-8f5e-556da5636cba)

I do not believe the issue stems from the docker-compose.yml configuration as I have tested various email and password lengths and different ports to no avail. But here it is:
![swetrix-config](https://github.com/Swetrix/roadmap/assets/22604901/a6cbf18e-24ca-44cc-bc89-44be37693d29) (port is now adjusted to 8080).

In summary, there appears to be no way to access the interface using the current Docker container. While HTTP access to the interface would be acceptable, tracking would not function properly on unencrypted connections as modern browsers reject HTTP, especially if the rest of the site is HTTPS (which all are nowadays).

My key questions are:

1)    How can I configure SSL/HTTPS with a valid certificate to establish a connection?

2)    What steps can I take to resolve the login error?

Thank you in advance for your guidance.

**Comment by Blaumaus** [comment]
Hey! May you try to replace API_URL port to 8080 (i.e. http://85.215.231.202:8080)?

This variable is passed to the frontend of Swetrix and means the address of the API to send requests to

**Comment by Blaumaus** [comment]
for anyone reading this: the issue is solved. for some reason `docker-compose up -d` did not apply changed variables in the docker compose config.

running the `docker-compose down` and then `docker-compose up -d` again fixed it

**Comment by dabugen** [comment]
Thanks for all the help this evening! I really appreciate it. I am still wondering how SSL (HTTPS) could be used though, as this is pretty mandatory nowadays. Thank you.
